src/components/Form.js

```diff
@@ -20,24 +20,26 @@ const Form = () => {
 
     const bookObj = {
       ...book,
+      category: 'Action',
       item_id: Date.now().toString(),
     };
 
     dispatch(addBookAction(bookObj));
 
-    // clear values
     setBook({
       title: '',
       author: '',
     });
   };
 
   return (
-    <form method="POST" action="/" className="add">
+    <form method="POST" action="/" className="add" onSubmit={(e) => submitForm(e)}>
       <h3>ADD NEW BOOK</h3>
-      <input type="text" name="title" id="title" value={book.title} placeholder="Book title" onChange={handleChange} required />
-      <input type="text" name="author" id="author" value={book.author} placeholder="Author" onChange={handleChange} required />
-      <button type="submit" onClick={(e) => submitForm(e)}>Add Book</button>
+      <fieldset>
+        <input type="text" name="title" id="title" value={book.title} placeholder="Book title" onChange={handleChange} required />
+        <input type="text" name="author" id="author" value={book.author} placeholder="Author" onChange={handleChange} required />
+        <button type="submit">ADD BOOK</button>
+      </fieldset>
```

Comment:
- [ ] Please, consider that you have to add also the possibility to choose the Category for a new book. Here, in your version, any new book adds with a Category "Action" by default. I know you can fix it easily 👍  
<img width="1396" alt="Screen Shot 2022-04-03 at 00 39 19" src="https://user-images.githubusercontent.com/49064106/161400510-d3e6d004-b221-4ae6-ab74-95812bf9071c.png">